### PR TITLE
:sparkles: Added new rpc endpoint for combined files

### PR DESCRIPTION
**Pull request by zwaans**
# Changes
- deprecated get and get by id endpoints
- removed post endpoint from combined files (can't have duplicate keys in the same file)
- added post RPC endpoint

### CODE REVIEW

File: specification/paths/CombinedFiles.json [hunk 1]
```diff
@@ -76,7 +77,7 @@
   },
   "post": {
     "tags": [
-      "CombinedFiles"
+      "RPC"
```
Comment (M4tini) [inline specification/paths/CombinedFiles.json:80]: I think we need to keep this (while our app still uses it) and we need to introduce the RPC ass a new endpoint.
Since the RPC format is always `<verb>-<noun>` it can be `POST /combine-files` so it's not a duplicate route.

File: specification/schemas/CombinedFile.json [hunk 2]
```diff
@@ -50,7 +51,6 @@
         "relationships": {
           "type": "object",
           "required": [
-            "shop",
```
Comment (M4tini) [inline specification/schemas/CombinedFile.json:53]: Why was this removed?

File: specification/paths/CombinedFiles.json [hunk 3]
```diff
@@ -121,20 +144,12 @@
     },
     "responses": {
       "201": {
```
Comment (M4tini) [inline specification/paths/CombinedFiles.json:146]: For the new RPC endpoint, let's make this `200` like the others that return base64 data.

**Review by M4tini — CHANGES_REQUESTED** [review]

**Review by M4tini — APPROVED** [review]

File: specification/paths/CombineFiles.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            }
+          }
+        }
+      }
+    },
+    "responses": {
+      "200": {
+        "description": "Created the combined file.",
+        "content": {
+          "text/csv": {
```
Comment (M4tini) [inline specification/paths/CombineFiles.json:73]: I think this should be `application/pdf`. And I believe `application/zpl` support will be added in the near future, so we could add that as a possible "content" as well, but maybe that's too soon :)

**Review by M4tini — CHANGES_REQUESTED** [review]

File: specification/schemas/CombinedFile.json (hunk 2 above)
Comment (NickVries) [reply specification/schemas/CombinedFile.json]: Is there a good reason to keep the shop relationship?

Comment (M4tini) [reply specification/schemas/CombinedFile.json]: This schema will not exist anymore once the new RPC endpoint is done, so no need to change anything here.

**Review by M4tini — APPROVED** [review]
